src/components/stats/input.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import {
+  unstable_LowPriority as LowPriority,
+  unstable_runWithPriority as runWithPriority,
+  unstable_scheduleCallback as scheduleCallback
+} from "scheduler";
+
+const InputContainer = styled.p`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const InputTag = styled.input`
+  font-size: 24px;
+  border: 2px solid black;
+  border-radius: 3px;
+  width: 300px;
+  text-align: center;
+  margin: 10px;
+`;
+
+const Input = ({ async, onChange }) => {
+  const [count, setCount] = useState(10);
+
+  useEffect(() => onChange(count), []);
```

Comment:
Shouldn't you pass the count of state in the dependencies of the effect? I'm not sure about this comment

Reply:
This one made me think... This `useEffect` is used to initialize the parent with a count value because it's here, in the child that the real value of count is handled. If I add `count` in the array, it will trigger the parent callback on each change, it's something I do manually in the changeHandler below. So, as is, no it will not work but it could be done moving some of the logic from the handler to the effect. I'm not really sure which one is the "best".

Reply:
Just wanted to check if my understanding of this hook is good or not :p . Anyway, if it needs to move a lot of logic, I think you don't really have to care by now. WYT?

Reply:
I think it could stay as it is for now. But I seen eslint rules popping to enforce good practices around hooks. I wonder if this code will not trigger a warning.